src/Umbraco.Web/Editors/ContentController.cs

```diff
@@ -493,13 +494,19 @@ public HttpResponseMessage GetNiceUrl(Udi id)
                 Mapper.Map<IContent, ContentItemBasic<ContentPropertyBasic>>(content,
                     opts =>
                     {
+                        if(string.IsNullOrEmpty(cultureName) == false)
```

Comment:
Don't think you need to check for null/empty here, just assign it as-is and the rest of the logic will check for null/empy